newsletter_automation/newsletter/edit_article_form.py

```diff
@@ -22,6 +22,6 @@ class EditArticlesForm(FlaskForm):
     url= TextField('url', validators= [DataRequired()])
     title = TextField('title', validators= [DataRequired()])
     description = TextAreaField('description', validators= [ DataRequired(), Length(min=4)])
-    time = TextField('time',validators=[ DataRequired()])
+    time = IntegerField('time',validators=[NumberRange(min=1, max=60, message="Please provide a valid number")])
```

Comment:
Is the reading time limited to 60 mins here? If yes we need a broader range may be at least 3 characters

Reply:
Reading time now increased to 120 minute now.